SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h

```diff
@@ -403,7 +404,7 @@ extern const TArray<Worker_ComponentId> KnownEntityAuthorityComponents;
 //
 
 constexpr uint32 SPATIAL_SNAPSHOT_SCHEMA_HASH = 679237978;
-constexpr uint32 SPATIAL_SNAPSHOT_VERSION_INC = 2;
+constexpr uint32 SPATIAL_SNAPSHOT_VERSION_INC = 3;
```

Comment:
I think any schema files that get saved in the snapshot should be parsed via `GDKSchemaFilePaths` in `GIVEN_snapshot_affecting_schema_files_WHEN_hash_of_file_contents_is_generated_THEN_hash_matches_expected_snapshot_version_hash`. Which will then result in a change to the hash. cc: @martin-improbable

Reply:
Ah, no new components have been added